### PR TITLE
feat: add new config command for show and switch between profiles

### PR DESCRIPTION
**Pull request by nic-6443**

### CODE REVIEW

File: cmd/config/cmd.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+	"github.com/api7/cloud-cli/internal/output"
+	"github.com/api7/cloud-cli/internal/persistence"
+)
+
+// NewCommand creates the config sub-command object.
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "config [COMMAND] [ARGS...]",
+		Short: "Show or change configuration of the CLI",
```
Comment (tokers) [inline cmd/config/cmd.go:28]:
```suggestion
		Short: "Configuration management",
```

File: cmd/config/switch.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/api7/cloud-cli/internal/output"
+	"github.com/api7/cloud-cli/internal/persistence"
+)
+
+func newSwitchCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "switch",
+		Short:   "Switch the Profile used by Cloud CLI by default",
```
Comment (tokers) [inline cmd/config/switch.go:27]:
```suggestion
		Short:   "Switch the default profile used by Cloud CLI",
```

File: cmd/config/view.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+				return
+			}
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			config, err := persistence.LoadConfiguration()
+			if err != nil {
+				output.Errorf(err.Error())
+			}
+
+			if options.Global.Config.View.JSON {
```
Comment (tokers) [inline cmd/config/view.go:49]: Don't need the JSON.

File: docs/configuring-cloud-cli.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ After executing the `cloud-cli configure` command, cloud-cli will prompt you to
 enter the Personal Access Token.
 
 ```shell
-cloud-cli configure
+$ cloud-cli configure
```
Comment (tokers) [inline docs/configuring-cloud-cli.md:55]: Remove the $.

File: docs/configuring-cloud-cli.md [hunk 5]
```diff
@@ -84,7 +84,34 @@ take effect, if you want it to take effect immediately after creation, you can u
...
+we can use the `cloud-cli config view` command to see the full list of available profiles.
+
+```shell
+$ cloud-cli config view
++--------------+----------------+---------------+------------+----------------------------------+
+| PROFILE NAME |  ORGANIZATION  | CONTROL PLANE | IS DEFAULT |        API7 CLOUD ADDRESS        |
++--------------+----------------+---------------+------------+----------------------------------+
+| dev          | API7 dev       | default       | False      | https://dev.api7.cloud           |
+| eu           | API7 eu center | default       | False      | https://api.api7.cloud           |
+| aliyun       | API7 aliyun    | default       | False      | https://api.aliyun-hk.api7.cloud |
```
Comment (tokers) [inline docs/configuring-cloud-cli.md:113]: No default profile?

**Review by mscb402 — APPROVED** [review]

File: cmd/config/view.go (hunk 3 above)
Comment (nic-6443) [reply cmd/config/view.go]: Removed.

File: docs/configuring-cloud-cli.md (hunk 4 above)
Comment (nic-6443) [reply docs/configuring-cloud-cli.md]: Removed.

File: docs/configuring-cloud-cli.md (hunk 5 above)
Comment (nic-6443) [reply docs/configuring-cloud-cli.md]: Fixed.

**Review by tokers — APPROVED** [review]

**Pull request merged**